libs/elements/layout/texts/src/lib/elewa-text-content-item/elewa-text-content-item.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+import { ContentText } from '@elewa-website/models/schema/ui/texts';
+@Component({
+  selector: 'elewa-website-elewa-text-content-item',
+  templateUrl: './elewa-text-content-item.component.html',
+  styleUrls: ['./elewa-text-content-item.component.scss'],
+})
+export class ElewaTextContentItemComponent {
+  @Input() content!: ContentText;
```

Comment:
Good job for following the task description. 
Also, add comments describing what the input is for.